### PR TITLE
Reformatted kube-burner template

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Changes can be seen live here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/kube-burner/build?delay=0sec

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ pipeline {
   parameters {
         string(name: 'BUILD_NUMBER', defaultValue: '', description: 'Build number of job that has installed the cluster.')
         choice(choices: ["cluster-density","node-density","node-density-heavy","pod-density","pod-density-heavy","max-namespaces","max-services", "concurrent-builds"], name: 'WORKLOAD', description: '''Type of kube-burner job to run''')
-        booleanParam(name: 'WRITE_TO_FILE', defaultValue: false, description: 'Value to write to google sheet (will run https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/write-scale-ci-results)')
+        booleanParam(name: 'WRITE_TO_FILE', defaultValue: false, description: 'Value to write to google sheet (will run <a href=https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/write-scale-ci-results>write-scale-ci-results</a>)')
         booleanParam(name: 'CLEANUP', defaultValue: false, description: 'Cleanup namespaces (and all sub-objects) created from workload')
```
Comment (paigerube14) [inline Jenkinsfile:20]: Can you also add links to the cleanup and cerberus jobs for there parameters listed here

Comment (paigerube14) [reply Jenkinsfile:20]: And scaleup as well

File: Jenkinsfile [hunk 2]
```diff
@@ -38,8 +38,7 @@ pipeline {
             max-services: This will export SERVICE_COUNT env variable; set to 200 * num_workers, work up to 250 * num_workers. Creates n-replicas of an application deployment (hello-openshift) and a service in a single namespace.  <br>
             node-density: This will export PODS_PER_NODE env variable; set to 200, work up to 250. Creates as many "sleep" pods as configured in this variable - existing number of pods on node. <br>
             node-density-heavy: This will export PODS_PER_NODE env variable; set to 200, work up to 250. Creates this number of applications proportional to the calculated number of pods / 2 <br>
-            Read here for detail of each variable: <br>
-            https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/README.md <br>
+            Read <a href=https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/README.md>here</a> for details about each variable
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply Jenkinsfile:41]: Up to you - I'm fine with either :+1:

File: Jenkinsfile (hunk 1 above)
Comment (nathan-weinberg) [reply Jenkinsfile:20]: Sure

File: Jenkinsfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (paigerube14) [reply Jenkinsfile:41]: Think lets do the ocp-qe repo's README. It links to the e2e-benchmark one as well

File: Jenkinsfile [hunk 3]
```diff
@@ -14,33 +14,57 @@ pipeline {
...
+          name: 'CLEANUP',
+          defaultValue: false,
+          description: 'Cleanup namespaces (and all sub-objects) created from workload (will run <a href=https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/benchmark-cleaner/>benchmark-cleaner</a>)'
+        )
+        booleanParam(
+          name: 'CERBERUS_CHECK',
+          defaultValue: false,
+          description: 'Check cluster health status pass (will run <a href=https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/cerberus/>cerberus</a>)'
+        )
+        string(
```
Comment (paigerube14) [inline Jenkinsfile:42]: let's move this parameter down, right above ENV_VARS so we can keep the variables and workload parameter closter together

**Pull request merged**